### ISSUE TITLE
DataPower can't use "app" user

### ISSUE THREAD

**Issue by QAnders** [ibm-messaging/mq-docker#103]
I am trying to setup DataPower and MQ as Docker containers but I am having problems getting Datapower to connect to the MQ server. I get a 2035 Not authorized (Connect) error.
In Datapower you can't (for some strange reason) set the MCA user "app" so I've tried to modify the channel to use no auth but it doesn't seem to be working. 

As this is a locally run development only I'd like to remove all channel auth (if possible).

**Comment by arthurbarr** [comment]
Although it's not recommended, you can turn off channel auth as described [here](https://www.ibm.com/developerworks/community/blogs/messaging/entry/getting_going_without_turning_off_mq_security?lang=en)

**Comment by parrobe** [comment]
Hi @QAnders - What channel are you connecting via? The `DEV.ADMIN.SVRCONN`  is set to block any clients that connect but do not supply the admin user credentials. The `DEV.APP.SVRCONN` channel has it's `MCAUSER` attribute set to `app` so if you connected to this channel it should let you connect and set the MCAUSER to app.

If you've created your own channel then you will need to allow access to it by creating an addressmap channel authentication rule for that channel. You can do this by executing the following MQSC command:

`SET CHLAUTH('<CHANNEL NAME>') TYPE(ADDRESSMAP) ADDRESS('*') USERSRC(CHANNEL) CHCKCLNT(ASQMGR) ACTION(REPLACE)`

`

**Comment by QAnders** [comment]
Thanks @parrobe!
I've created my own channel (DATAPOWER.SVRCONN) that has no MCAUSER.
After I found the command to be able to run MQ Script all went a bit easier... :)

`docker exec --tty --interactive mq01 runmqsc`

Then I ran:
```
ALTER QMGR CHLAUTH(DISABLED) CONNAUTH(' ')
REFRESH SECURITY TYPE(CONNAUTH)
ALTER CHANNEL('DATAPOWER') CHLTYPE(SVRCONN) MCAUSER('mqm')
```

After this I am able to connect both via RFHUTIL and DataPower using no MCAUSER or user name/password (which suits me fine).
